app/models/concerns/orchestration/compute.rb

```diff
@@ -67,7 +68,11 @@ def queue_compute_create
   end
 
   def queue_compute_update
-    return unless compute_update_required?
+    unless compute_update_required?
+      logger.error "No compute update required."
```

Comment:
This should be really a debug message as this can be often the case.

Reply:
Works fine also with `logger.debug`.